static/app/views/dashboards/datasetConfig/utils.tsx

```diff
@@ -0,0 +1,111 @@
+import trimStart from 'lodash/trimStart';
+
+import type {EventsStatsOptions} from 'sentry/actionCreators/events';
+import type {PageFilters} from 'sentry/types/core';
+import type {Organization} from 'sentry/types/organization';
+import {isEquation, isEquationAlias} from 'sentry/utils/discover/fields';
+import type {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {TOP_N} from 'sentry/utils/discover/types';
+import {DisplayType, type Widget} from 'sentry/views/dashboards/types';
+import {getNumEquations, getWidgetInterval} from 'sentry/views/dashboards/utils';
+
+// This function is used to generate the request data for timeseries widgets
+// and is intended to be used eventually by doEventsRequest
```

Comment:
```suggestion
/**
 * This function is used to generate the request data for timeseries
 widgets and is intended to be used eventually by doEventsRequest
 */
```

Multi-line docs comments show up better in autocomplete and type hints

<img width="403" alt="Screenshot 2024-11-04 at 10 07 27 AM" src="https://github.com/user-attachments/assets/9bbe0866-3f9d-47f4-8007-62c463a65a4f">